src/Configuration/SdkConfiguration.php

```diff
@@ -141,7 +141,7 @@ public function __construct(
         $this->setupStateCookies();
         $this->setupStateFactories();
 
-        if (in_array($this->getStrategy(), self::STRATEGIES_USING_SESSIONS, true)) {
+        if ($this->usingStatefulness()) {
```

Comment:
This logic has been moved into the new `usingStatefuless()` method for reusability.